src/lib/dev-environment/types.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 export interface InstanceOptions {
-	title: string;
-	multisite: boolean;
+	title?: string;
```

Comment:
Changed these `InstanceOptions` properties to optional since they [can be missing and are treated as optional by the code](https://github.com/Automattic/vip-cli/blob/4de9a92/src/lib/dev-environment/dev-environment-cli.js#L277-L278). This made merging with `ConfigurationFileOptions` easier.